.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
I think we can generalise and run our jobs for every branch and not only this one.

Reply:
Sure! So every push and every pull request? – Will change it.

Reply:
Yes please :)